docs/doc/100-faq/30-license-faqs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Licensing FAQs
+---
+
+Databend code is licensed in two ways:
+
+* Apache 2.0 License (Apache)
+* Elastic 2.0 License (Elastic)
+
+Databend core is free to use. Most of the core features are licensed under the Apache License, while 
+features under `src/query/ee` and `src/meta/ee` directory are subject to the Elastic License.
+
+Databend Enterprise features require a [paid license](#obtain-a-license) from databend and are licensed under the Elastic License.
+
+:::note
+You can find the feature's license by taking a look on the code file's header under the [Databend repository](https://github.com/datafuselabs/databend)
+:::
+
+## Types of licenses
+
+| Type                | Description                                                                                                                                                                                                                                                                                         |
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Apache 2.0 License  | Core features under the Apache License are free to use and fully open source.                                                                                                                                                                                                                       |
+| Elastic 2.0 License | * Elastic (Free) features are free to use. The source code is available to view and modify under Elastic 2.0 License Limitations  <br> * Elastic (Paid) features require an Enterprise License key to access. The source code is available to view and modify under Elastic 2.0 License Limitations | 
```

Comment:
It is confusing here. It feels like a distinction needs to be made between Elastic features and Enterprise features. In particular, we mentioned earlier that Databend Enterprise features require a paid license; here, it is noted that we may have some Elastic free features.

Reply:
Elastic License could provide source avaialble free features, we just need to ensure that all enterprise features are proteced by license key